packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -30,7 +33,102 @@ class Console extends StatefulWidget {
 }
 
 class _ConsoleState extends State<Console> {
-  final scrollController = ScrollController();
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<List<String>>(
+      valueListenable: widget.controller.stdio,
+      builder: (context, lines, _) {
+        // Extract this to a reusable ConsoleWithControls StatelessWidget?
+        return Stack(
+          children: [
+            ConsoleOutput(lines: lines),
+            Container(
+              alignment: Alignment.bottomRight,
+              child: ConsoleControls(
+                disabled: lines.isEmpty,
+                onCopyPressed: () {
+                  Clipboard.setData(ClipboardData(
+                    text: lines.join('\n'),
+                  )).then((_) {
+                    Notifications.of(context)?.push(
+                      'Copied!',
```

Comment:
Perhaps, `Copied n lines.` to give the user a bit more info? `Copied nnn characters.`? I you change the text, we have some simple utility methods to help with pluralization.

Reply:
@devoncarew I did a quick search and I couldn't find much i18n, but I can change this to `Copied n lines.`. Where's the pluralization stuff?

Reply:
If you search through the source for `pluralize` you'll see it - it'll look something like this: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart#L380

Reply:
(lib/src/utils.dart -> pluralize)